src/modules/linkdrop-p2p/index.ts

```diff
@@ -156,6 +168,18 @@ class LinkdropP2P implements ILinkdropP2P {
     }
   }
 
+
+  getVersionFromEscrowContract: TGetVersionFromEscrowContract = (escrowAddress) => {
+    const escrowVersions = Object.keys(escrows)
+    const result = escrowVersions.find(version => {
+      const escrowsForVersion = escrows[version]
+      if (escrowsForVersion && escrowsForVersion.length > 0) {
+        return escrowsForVersion.find(item => item.toLowerCase() === escrowAddress.toLowerCase())
+      }
+    })
+    return result || null
```

Comment:
raise an explicit error if the version for the provided `escrowAddress` is not found

Reply:
throw new Error("Version not found for the provided escrow contract address")